### PR TITLE
add new description

### PR DESCRIPTION
**Pull request by yaigrie**

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -86,7 +89,8 @@
         let result = await openModal(FooComponent);
         console.log(result); // Whatever the modal returned when it was closed
       }
-    <`+dedent`/script>
+    <` +
+      dedent`/script>
```
Comment (zeppelin) [inline src/routes/+page.svelte:93]: Is this a neccessary change?

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -29,9 +29,12 @@
 <svelte:head>
   <title>Svelte Promise Modals</title>
   <meta property="og:title" content="Svelte Promise Modals" />
-  <meta property="og:site_name" content="Svelte Promise Modals" />
+  <meta property="og:site_name" content="Svelte Promise Modals - Modals made easy." />
```
Comment (zeppelin) [inline src/routes/+page.svelte:32]: Wait, the site name was not OK with LI?

**Review by zeppelin — CHANGES_REQUESTED** [review]

File: src/routes/+page.svelte (hunk 1 above)
Comment (yaigrie) [reply src/routes/+page.svelte:93]: ah no sry, this is a prettier change

File: src/routes/+page.svelte (hunk 2 above)
Comment (yaigrie) [reply src/routes/+page.svelte:32]: The one tool said this:  og:title metatag found
 Your title should be between 30-60 characters, with a maximum of 90 (currently 21 characters)

**Comment by yaigrie** [comment]
I found one site saying that you should try 600px height instead of 630, let's try that

**Review by zeppelin — APPROVED** [review]

**Pull request merged**